src/components/CreateSecret/Form/index.js

```diff
@@ -0,0 +1,159 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { injectIntl } from 'react-intl';
+
+import { getErrorMessage } from '@tektoncd/dashboard-utils';
+import { Button, InlineNotification } from 'carbon-components-react';
+import UniversalFields from './UniversalFields';
+import PasswordType from './PasswordType';
+import AccessTokenType from './AccessTokenType';
+import Annotations from './Annotations';
+
+function getNotification(message, intl, callback) {
+  const notification = (
+    <InlineNotification
+      kind="error"
+      title={intl.formatMessage({
+        id: 'dashboard.error.title',
+        defaultMessage: 'Error:'
+      })}
+      subtitle={message}
+      iconDescription={intl.formatMessage({
+        id: 'dashboard.notification.clear',
+        defaultMessage: 'Clear Notification'
+      })}
+      className="notificationComponent"
+      data-testid="errorNotificationComponent"
+      lowContrast
+      onCloseButtonClick={callback}
+    />
+  );
+  return notification;
+}
+
+const Form = props => {
+  const {
+    annotations,
+    name,
+    namespace,
+    accessToken,
+    username,
+    password,
+    invalidFields,
+    secretType,
+    errorMessageDuplicate,
+    loading,
+    handleClose,
+    intl,
+    submit,
+    handleChangeTextInput,
+    handleChangeNamespace,
+    handleSecretType,
+    handleAnnotationChange,
+    handleAdd,
+    handleRemove,
+    errorMessageCreated,
+    removeDuplicateErrorMessage
+  } = props;
+
+  return (
+    <div className="createSecret">
+      {errorMessageDuplicate &&
+        getNotification(
+          errorMessageDuplicate,
+          intl,
+          removeDuplicateErrorMessage
+        )}
+      {errorMessageCreated &&
+        getNotification(getErrorMessage(errorMessageCreated), intl, null)}
+      <div className="heading">
+        <h1>
+          {intl.formatMessage({
+            id: 'dashboard.createSecret.heading',
+            defaultMessage: 'Create new Secret'
+          })}
+        </h1>
+        <Button
+          iconDescription={intl.formatMessage({
+            id: 'dashboard.createSecret.cancelButton',
```

Comment:
```suggestion
            id: 'dashboard.modal.cancelButton',
```

This is the same string we use for cancel on all forms / modals to help us stay consistent.

same for other instances of 'cancel' below